input/pagecontent/2-3-Events.md

```diff
@@ -35,16 +35,17 @@ EventName ::= (FHIRresource | '*') ('-') ( 'open' | 'close' | 'update' | 'select
 
 The `FHIRresource` indicates the focus of the event; the `suffix` defines the type of event.
 
-Event names are unique and case-insensitive. Implementers may define their own events. Such proprietary events SHALL be named with reverse domain notation (e.g. `org.example.patient_transmogrify`). Reverse domain notation SHALL NOT be used by a standard event catalog. Proprietary events SHALL NOT contain a dash ("-").
+Event names are unique and case-insensitive. It is RECOMMENDED to use [Upper-Camel](https://en.wikipedia.org/wiki/Camel_case) case.
 
-When subscribing to FHIRcast events a list of events is added. These events may contain wild cards. Wild cards are expressed as a `*` replacing either the `FHIRresource` or `suffix`  with `*` indicates any events that match the resulting definition are requested. The event `*` means the subscriber subscribes to any event. The table below shows some typical examples.
+Implementers may define their own events. Such proprietary events SHALL be named with reverse domain notation (e.g. `org.example.patient_transmogrify`). Reverse domain notation SHALL NOT be used by a standard event catalog. Proprietary events SHALL NOT contain a dash ("-").

```

Comment:
Will the reverse domain notation conflict with the -forward proposal which should be coming from Isaac?  I guess we can adjudicate then.